Dockerfile

```diff
@@ -1,5 +1,4 @@
-ARG BASE_IMAGE_TAG=focal-3.8
-FROM makinacorpus/geodjango:${BASE_IMAGE_TAG}
+FROM makinacorpus/geodjango:jammy-3.10
```

Comment:
The base image is hardcoded to `makinacorpus/geodjango:jammy-3.10`. It would be more flexible to use a build argument for the base image, allowing the Dockerfile to be used with different base images without modification. [medium]